co2eq/purchase/index.js

```diff
@@ -125,6 +125,10 @@ function conversionCPI(eurAmount, referenceYear, countryCodeISO2, datetime) {
     return eurAmount;
   }
 
+  if (!referenceYear) {
```

Comment:
@pierresegonne I had to move the check inside the conversion function to make sure it doesn't trigger for cases that don't require conversions.

Reply:
That makes sense 👍